openquakeplatform/openquakeplatform/exposure/static/exposure/exposure_controller.js

```diff
@@ -71,10 +71,25 @@ var showErrorDialog = function(message, options) {
     $("#error-dialog").dialog(options);
 };
 
-app.controller('ExposureCountryList', function($scope, $filter, myService, ngTableParams) {
+app.controller('ExposureCountryList', function($scope, $filter, myService, ngTableParams, $http) {
+
+    // Set up the templates
+    $scope.templates = [
+    {
```

Comment:
Why, instead of use complex template approach use a more simple `ng-show` directive with different comparisons ?
In this way all the html code will be in the same place but just what you want to show will be visualized.
Here a simple example: http://jsfiddle.net/nastasi/g4mfz5c2/